### PR TITLE
Remove blobdb export toggle

### PR DESCRIPTION
**Pull request by benrudolph**
anyone

### CODE REVIEW

File: corehq/apps/export/tasks.py [hunk 1]
```diff
@@ -28,30 +27,17 @@ def populate_export_download_task(export_instances, filters, download_id, filena
...
-            expose_cached_download(
-                payload,
-                expiry,
-                ".{}".format(file_format.extension),
-                mimetype=file_format.mimetype,
-                content_disposition=safe_filename_header(filename, file_format.extension),
-                download_id=download_id,
-            )
+        db = get_blob_db()
+        db.put(open(export_file.path, 'r'), download_id, timeout=expiry)
```
Comment (millerdev) [inline corehq/apps/export/tasks.py]: I realize this didn't change here, but would be good to close the file:
```py
with open(export_file.path, 'r') as file_:
    db.put(file_, download_id, timeout=expiry)
```

**Review by millerdev — APPROVED** [review]
one nitpicky comment

**Comment by calellowitz** [comment]
👍 pending daniel's comment

**Comment by benrudolph** [comment]
good call will update

**Comment by benrudolph** [comment]
ok refactored a little to get rid of a confusing layer of indirection

**Pull request merged**